test/src/main/java/org/apache/accumulo/test/functional/DurabilityIT.java

```diff
@@ -36,17 +29,22 @@
 import org.apache.accumulo.miniclusterImpl.MiniAccumuloConfigImpl;
 import org.apache.accumulo.miniclusterImpl.ProcessReference;
 import org.apache.accumulo.test.categories.MiniClusterOnlyTests;
-import org.apache.accumulo.test.categories.PerformanceTests;
 import org.apache.accumulo.test.mrit.IntegrationTestMapReduce;
 import org.apache.hadoop.conf.Configuration;
 import org.apache.hadoop.fs.RawLocalFileSystem;
 import org.junit.BeforeClass;
 import org.junit.Test;
 import org.junit.experimental.categories.Category;
 
-import com.google.common.collect.Iterators;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Map.Entry;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assume.assumeFalse;
 
-@Category({MiniClusterOnlyTests.class, PerformanceTests.class})
```

Comment:
Can probably remove the `PerformanceTests.java` file, too.

Reply:
@ctubbsii  I can do that. Would you like `ContinuousIngest.java` to be removed as well?

Reply:
Yes, I believe that can be safely removed also, but might want to check that nothing in accumulo-testing uses it. I believe it has its own copy anyway.

Reply:
Nothing in accumulo-testing showed any usage of the `ContinuousIngest.java` from accumulo, and there is indeed a separate copy of the class in accumulo-testing. I will also remove `ContinousOpts.java` since it was only used by `ContinousIngest.java`.